src/app/admin/AdminPage.tsx

```diff
@@ -23,6 +32,7 @@ interface Panel {
 
 interface Props {
     userData: UserRow[];
+    collectionCentreData: Schema["collection_centres"][];
```

Comment:
See later reply, think this should be explicit!

Reply:
In this case, it doesn't matter if there is any schema change in the database table, as all I want to do is to display all the content in the database in table form
I am not using anything specific to e.g. string to do specific operation for this